### DISCUSSION TITLE
How do you cut extrude in fornjot ?

### DISCUSSION THREAD

**Discussion by Shrav108** [hannobraun/fornjot#2451] [Q&A]
What is the method of cut extruding in fornjot ?

**Comment by hannobraun** [comment]
It's probably not possible right now. See https://github.com/hannobraun/fornjot/issues/2264 for some context.

_If_ it is possible, it's going to be limited to cases where there are no intersections between the extruded hole and existing geometry.

I'm converting this into a discussion, as that's a more suitable venue for answering questions.

**Comment by Shrav108** [comment]
Well, I guess I have to wait till this crate gets better. I was trying to integrate fornjot into an app ( egui ) where one can generate CAD file directly after some calculations.
